tests/performance/live.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+MARATHON_DIR=$(pwd)
+MARATHON_PERF_TESTING_DIR=$(pwd)/marathon-perf-testing
+
+# Bring an up-to-date marathon-perf-testing environment
+[ ! -d marathon-perf-testing ] && git clone "https://${GIT_USER}:${GIT_PASS}@github.com/mesosphere/marathon-perf-testing.git"
+(cd marathon-perf-testing; git pull --rebase)
+
+# Privileged clean-up of the results folder
+docker run -i --rm \
+    --privileged \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v "$MARATHON_DIR:/marathon" \
+    icharalampidis/marathon-perf-testing:latest \
+    rm -rf /marathon/results
+
+# Configuration
+DOCKER_NETWORK=testing
+
+# Get the git hash
+GIT_HASH=$(cd "$MARATHON_DIR" && git rev-parse HEAD)
+echo "- Running on GIT hash $GIT_HASH..."
+
+# Run the CI job
+docker run -i --rm \
+    --network ${DOCKER_NETWORK} \
+    --privileged \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v "$MARATHON_DIR:/marathon" \
+    -e "PARTIAL_TESTS=test-5k-apps" \
+    -e "PERF_DRIVER_ENVIRONMENT=env-ci-live.yml" \
+    -e "DATADOG_API_KEY=$DATADOG_API_KEY" \
+    -e DCLUSTER_ARGS="--docker_network='${DOCKER_NETWORK}' --marathon_jmx_host=marathon_1 --share_folder=${MARATHON_PERF_TESTING_DIR}/files" \
```

Comment:
If you want to remove the `DATAOG_APP_KEY` you should also change:

1. https://github.com/mesosphere/marathon/blob/master/tests/performance/scripts/dcluster_run.sh#L25
2. https://github.com/mesosphere/marathon/blob/master/tests/performance/scripts/dcluster_run.sh#L61
3. https://github.com/mesosphere/marathon/blob/master/tests/performance/config/perf-driver/reporters/report-datadog.yml#L18

Otherwise you will get an error while starting.
Otherwise, keep the environment variable, and set it to an arbitrary value.

Reply:
Thanks for the pointers. I'm kind of surprised to see `live.sh` wrap `dcluster.sh`. Anyhow. I made the changes. I have to admit though that I don't understand why we wire the Python classes with YAML files instead of pure Python. What is the motivation?

Reply:
I'm wondering if we could go for a pure Python config but would like to understand the added value of the YAML configs first 🙂.